tests/HelpersTest.php

```diff
@@ -154,4 +155,14 @@ function tinker()
         $there = 'value';
         tinker('hey', $there);
     }
+
+    /* @test */
+    function dumpsql()
```

Comment:
Tests functions appear to be ordered alphabetically. Please move it to after the `faker` test for consistency.